integration-tests/cstor-pool/cstor_integration_test.go

```diff
@@ -0,0 +1,370 @@
+/*
+Copyright 2018 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cstorpoolit
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	csp "github.com/openebs/maya/pkg/cstorpool/v1alpha3"
+	spc "github.com/openebs/maya/pkg/storagepoolclaim/v1alpha1"
+	"time"
+
+	//	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"testing"
+)
+
+const (
+	KubeConfigPath = "/home/ashutosh/.kube/config"
+	MaxRetry       = 30
+)
+
+// client encapsulates various kubernetes object client set.
+type client struct {
+	cspClient *csp.Kubeclient
+	spcClient *spc.Kubeclient
+}
+
+// TestIntegrationCstorPool function instantiate the cstor pool test suite.
+func TestIntegrationCstorPool(t *testing.T) {
+	// RegisterFailHandler is used to register failed test cases and produce readable output.
+	RegisterFailHandler(Fail)
+	// RunSpecs runs all the test cases in the suite.
+	RunSpecs(t, "Cstor pool integration test suite")
+}
+
+var _ = Describe("STRIPED SPARSE SPC", func() {
+
+	var (
+		spcObj *apis.StoragePoolClaim
+	)
+	BeforeEach(func() {
+		spcObj = spc.NewSPCBuilder().
+			WithName("sparse-striped-auto").
+			WithDiskType(string(apis.TypeSparseCPV)).
+			WithMaxPool(3).
+			WithOverProvisioning(false).
+			WithPoolType(string(apis.PoolTypeStripedCPV)).
+			Build()
+	})
+
+	// Test Case #1 (sparse-striped-auto-spc). | TestType : Pool Creation
+	When("We apply sparse-striped-auto spc yaml with maxPool count equal to 3 on a k8s cluster having at least 3 capable node", func() {
+		It("pool resources count should be 3 with no error and healthy status", func() {
+			// Create a storage pool claim
+			_, err := clientInstance.spcClient.Create(spcObj)
+			Expect(err).To(BeNil())
+			cspCount := isHealthyCspCount(spcObj.Name, 3)
```

Comment:
`ops.csp.IsHealthyReplicas(spcObj.Name, 3)`

Reply:
Will fix it.